app/profile/routes.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint, request, jsonify
+from flask_jwt_extended import jwt_required
+from app.profile import controllers
+
+user_bp = Blueprint('user', __name__)
+
+@user_bp.route('/auth/profile', methods=['GET'])
+@jwt_required
+def user(user_email):
+    response, code = controllers.display(user_email)
```

Comment:
```suggestion
def user():
    user_id = get_jwt_identity()
    response, code = controllers.display(user_id)
```
since we r using user_id in the jwt